cardano-node/cardano-node.cabal

```diff
@@ -146,7 +146,7 @@ library
                       , async
                       , base16-bytestring
                       , bytestring
-                      , cardano-api ^>= 10.8
+                      , cardano-api ^>= 10.11.1
```

Comment:
Why `10.11.1` here vs `10.11` in cardano-testnet?

Reply:
10.11.1 should be enforced by index here anyway, so it could be 10.11. I've just added just in case for some reason is isn't. In cardano-testnet, 10.11.1 is enforced by dependency on cardano-cli.

Reply:
It doesn't matter much tbh - I could set it to 10.11.